suite-native/module-staking-management/src/components/StakingUnavailableBottomSheet.tsx

```diff
@@ -0,0 +1,61 @@
+import { BottomSheet, Button, Text } from '@suite-native/atoms';
+import { Translation } from '@suite-native/intl';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const bottomSheetElementStyle = prepareNativeStyle(utils => ({
+    marginVertical: utils.spacings.small,
+}));
+
+type StakingUnavailableBottomSheet = {
+    isCardSelected: boolean;
+    handleToggleBottomSheet: (value: boolean) => void;
+    handleDesktopClick: () => void;
+};
+
+export const StakingUnavailableBottomSheet = ({
+    isCardSelected,
+    handleToggleBottomSheet,
+    handleDesktopClick,
+}: StakingUnavailableBottomSheet) => {
+    const { applyStyle } = useNativeStyles();
+
+    if (!isCardSelected) return null;
+
+    return (
+        <BottomSheet
+            title={
+                <Text variant="titleSmall" textAlign="center">
+                    <Translation id="staking.stakingBottomSheet.title" />
+                </Text>
+            }
+            isVisible
+            isCloseDisplayed={false}
+            onClose={() => handleToggleBottomSheet(false)}
+            paddingHorizontal="large"
+        >
+            <Text
+                color="textSubdued"
+                textAlign="center"
+                style={applyStyle(bottomSheetElementStyle)}
+            >
+                <Translation id="staking.stakingBottomSheet.description" />
+            </Text>
+
+            <Button
+                onPress={() => handleToggleBottomSheet(false)}
```

Comment:
`handleToggleBottomSheet(true)` is never used in this file. The argument seems to be redundant in context of this file. What about passing `() => handleToggleBottomSheet(false)` from parent directly?
```suggestion
                onPress={handleToggleBottomSheet}
```

Reply:
good idea, fixed